src/devtools/client/debugger/src/actions/ui.js

```diff
@@ -100,14 +95,21 @@ export function showSource(cx, sourceId) {
       });
     }
 
-    // Make sure the sources panel is expanded so that the user
-    // sees the source.
     if (getSourcesCollapsed(getState())) {
       dispatch({
         type: "TOGGLE_SOURCES",
         sourcesCollapsed: false,
       });
     }
+  };
+}
+
+export function showSource(cx, sourceId) {
```

Comment:
I moved the panel opening logic away from `showSource` and into `ensureSourcesIsVisible`. This is because we don't always want to force-open the sources panel when we're showing the source in the editor. 

For example, when we rewind to a console message, we want to show that source in the editor, but it's more important to show the pause information panel. 

Instead, this makes it so we can just call `ensureSourcesIsVisible` where we need it (e.g. clicking a console message source link, clicking reveal in tree).